clover.rb

```diff
@@ -162,7 +173,9 @@ def self.freeze
       :verify_login_change, :change_password_notify, :confirm_password
     title_instance_variable :@page_title
 
+    # :nocov:
     unless Config.development?
+
```

Comment:
spurious, right?

Reply:
<img width="2416" alt="Screenshot 2023-06-14 at 18 51 05" src="https://github.com/ubicloud/ubicloud/assets/993199/d54e78f6-27d3-4169-9263-964278eab910">

Since it's test env, this block runs actually. But we couldn't test `else` branch without reloading file. I'm not sure how to exclude else part.
maybe?
```ruby
    unless Config.development?

      enable :disallow_common_passwords, :verify_account

      email_from Config.mail_from

      verify_account_view { view "auth/verify_account", "Verify Account" }
      resend_verify_account_view { view "auth/verify_account_resend", "Resend Verification" }
      verify_account_email_sent_redirect { login_route }
      verify_account_email_recently_sent_redirect { login_route }
      verify_account_set_password? false
    else
      # :nocov:
      # :nocov:
    end

```

Reply:
Oh I was thinkin it was an extra newline. No, it isn't?

Reply:
You are right 😅  I checked wrong lines